### ISSUE TITLE
Fix credits formatting

### ISSUE THREAD

**Issue by WildWeazel** [C7-Game/Prototype#176]
Make the credits screen use Noto Sans font with formatting for bold tags, etc.

**Comment by QuintillusCFC** [comment]
Note to future self if I don't finish investigating this tonight: The reason that bold isn't working is the note at https://docs.godotengine.org/en/stable/tutorials/ui/bbcode_in_richtextlabel.html that reads:

```
For BBCode tags such as [b] (bold), [i] (italics) or [code] to work, you must set up custom fonts for the RichTextLabel node first.

There are no BBCode tags to control vertical centering of text yet.
```

(the last part is why the centering isn't applied)

So it makes more sense to start with the font than the bold as I just did...

**Comment by QuintillusCFC** [comment]
Made some more progress on this tonight.

Downloaded bold/italic/bold+italic variants of Noto Sans via https://fonts.google.com/noto/specimen/Noto+Sans

Used a theme to set the bold/italic fonts on the rich text box, and figured out which properties I needed to update via https://github.com/godotengine/godot/pull/59013/files#diff-d6d99bb0566976eeb03e0f87cbd7f6a77c0904fbef222013eafbc567650105d7 .  Godot 4.0 will have simulated bold/italic font out of the box, but we don't want to be waiting for Godot 4.0 to reach full release, and even after that point, using real fonts will probably give us better results.  The 4.0 simulated-bold/italics looks blurry to me.

Also a general note, if we're struggling to figure out which properties we need to set programmatically on themes, the source lives at https://github.com/bruvzg/godot/blob/master/scene/resources/default_theme/default_theme.cpp .  Even though we're in Mono, the snake_case properties _are_ the ones we want to use in this case.  I've found the documentation for setting theme properties programmatically to be lacking, especially for C#/Mono, so this reference is likely to save time.

(I also plan to integrate the bold/italic/bold+italic options into the base theme, which may be done non-programmatically, but being a programmer I tend to gravitate towards programmatic solutions first)
